README.md

```diff
@@ -53,4 +53,10 @@ pm = currentProgram.treeManager.getRootModule(currentProgram.treeManager.DEFAULT
 # Relevant snippets of code
 `currentProgram.symbolTable.externalSymbols`
 
-`currentProgram.symbolTable.getAllSymbols(0)` [docs](https://ghidra.re/ghidra_docs/api/ghidra/program/model/symbol/SymbolTable.html#getAllSymbols(boolean))
\ No newline at end of file
+`currentProgram.symbolTable.getAllSymbols(0)` [docs](https://ghidra.re/ghidra_docs/api/ghidra/program/model/symbol/SymbolTable.html#getAllSymbols(boolean))
+
+# TO Dos
+- document how "*_data.csv" "*_symbols.csv" and "*.gf" are generated
+- find a way to programmatically generate "*.gf" files [related issue](https://github.com/NationalSecurityAgency/ghidra/issues/6640)
```

Comment:
The dump from the whole CFG of the program may be done either if we find some way to stitch CFG of the function together. Or it is said there is a script in the main repo. Maybe that can be of help.

Check this: https://github.com/NationalSecurityAgency/ghidra/issues/444

Reply:
I was thinking about that [here](https://github.com/MirciuPirciu/CFG_Malware_Script_Thingy/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R86)(`[...] document what information that functions uses from the dot files(we may gather that data from api)`)